client/verta/verta/credentials.py

```diff
@@ -67,11 +70,12 @@ def headers(self):
             # without underscore, for NGINX support
             # https://www.nginx.com/resources/wiki/start/topics/tutorials/config_pitfalls#missing-disappearing-http-headers
             "developer-key": self.dev_key,
+            "organization-id": self.organization_id or "",
```

Comment:
Would it be preferable to do a

```python
if self.organization_id:
    headers["organization-id"] = self.organization_id
```

instead of passing an empty string?

Reply:
Sounds good.